src/MovieRow.spec.js

```diff
@@ -2,9 +2,21 @@ import {render, screen} from '@testing-library/react';
 import MovieRow from './MovieRow';
 
 describe('MovieRow', () => {
-  it('ADD YOUR FIRST TEST HERE', () => {
-    // add your test here
+  it('component will display text', () => {
```

Comment:
Usually, when we write a test we start the description with "should...", read more here https://leanylabs.com/blog/good-unit-tests/